osmdroid-android/src/main/java/org/osmdroid/views/MapView.java

```diff
@@ -342,20 +339,35 @@ public Projection getProjection() {
 		return mProjection;
 	}
 
+	/**
+	 * Use {@link #resetProjection()} instead
```

Comment:
i'm a little confused by the javadoc

Reply:
The only use of `setProjection` was `setProjection(null)`. The actual use was to reset the projection, so that it gets recomputed later in `getProjection` (after a change of zoom level or center).